_components/character-count/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/character-count/accessibility-tests/
+layout: accessibility-test
+component:
+  name: character count
+title: Character count accessibility tests
+category: Components
+lead: Any USWDS character count component should pass these manual accessibility tests.
```

Comment:
I like the addition of "component here" as well. I went ahead and added it to the process doc for this section. 

It looks like other accessibility test pages just say the component name here. Should we include this change in #2581 since all of the accessibility test pages are changed there?

ex:
https://github.com/uswds/uswds-site/blob/491041dc07cfea2c54e21ce587bfab6c6b935431/_components/button/accessibility-tests.md?plain=1#L8